atmo/jobs/models.py

```diff
@@ -38,6 +38,10 @@ class SparkJob(EMRReleaseModel, CreatedByModel):
         help_text="Job name, used to uniqely identify individual jobs.",
         unique=True,
     )
+    description = models.TextField(
+        help_text='Job description.',
+        default='',
```

Comment:
That default is needed for the migration for existing records in the database. Is there a better place for that to live?